### DISCUSSION TITLE
Set custom fields when creating a new issue

### DISCUSSION THREAD

**Discussion by danjee** [ankitpokhrel/jira-cli#274] [Q&A] [answered]
Hello and thanks again fot this nice project. I haven't found a way to set a custom field or o template sample in the documtation. Is there something like 
`issue create -p PRJ -cf field_xxx="Value"` available? 
Also it will be nice to have a template usage sample and how to replace values on the template.

Thank you!

**Comment by ankitpokhrel** [comment]
Hi @danjee, setting custom field when creating the issue is not possible at the moment. There is already an open request for this: https://github.com/ankitpokhrel/jira-cli/discussions/222. It is bit tricky but I will see what I can do here when I get some time.

>  Also it will be nice to have a template usage sample and how to replace values on the template.

I will try to clarify this in the documentation.

**Reply by danjee** [reply]
Thanks for the information

**Comment by ankitpokhrel** [comment, marked as answer]
This feature is available since [v1.0.0-beta](https://github.com/ankitpokhrel/jira-cli/releases/tag/v1.0.0-beta)
